suricata.yaml.in

```diff
@@ -165,6 +165,10 @@ outputs:
             # Enable the logging of tagged packets for rules using the
             # "tag" keyword.
             tagged-packets: yes
+        # app layer records
+        - record:
+            # TODO disable by default. Too noisy.
+            enabled: yes
```

Comment:
Not really specific to this, but I wonder if we should name it "debug:record" (or `debug-record` if the `:` doesn't work), just to make it clear.  We'll come across this case again where an output is useful for debugging/dev, but really shouldn't be turned on by most end-users in production.

Reply:
Perhaps ya, though I'd also see it as useful for rule writers/analysts, so the name shouldn't be too dev-y